OpenMarket/OpenMarket/View/ProductGridCell.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  ProductGridCell.swift
+//  OpenMarket
+//
+//  Created by papri, Tiana on 17/05/2022.
+//
+
+import UIKit
+
+class ProductGridCell: UICollectionViewCell {
+    static let reuseIdentifier = "product-grid-cell-reuse-Identifier"
+    let cellUIComponent = CellUIComponent()
+    private var item: Product? = nil
+    
+    //MARK: - stackView
+    private lazy var baseStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.alignment = .center
+        stackView.spacing = 2
+        return stackView
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        addSubViews()
+        layout()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension ProductGridCell {
+    private func addSubViews() {
+        contentView.layer.borderWidth = CGFloat(1)
+        contentView.layer.cornerRadius = CGFloat(10)
+        contentView.layer.borderColor = UIColor.systemGray.cgColor
+        contentView.addSubview(baseStackView)
+        baseStackView.addArrangedSubviews(cellUIComponent.thumbnailImageView, cellUIComponent.nameLabel, cellUIComponent.priceLabel, cellUIComponent.bargainPriceLabel, cellUIComponent.stockLabel)
+    }
+    
+    private func layout() {
+        let thumbnail = cellUIComponent.thumbnailImageView
+        
+        let inset = CGFloat(10)
+        NSLayoutConstraint.activate([
+            baseStackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: inset),
+            baseStackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -inset),
+            baseStackView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: inset),
+            baseStackView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -inset)
+        ])
+        
+        NSLayoutConstraint.activate([
+            thumbnail.widthAnchor.constraint(equalToConstant: contentView.frame.width * 0.7),
+            thumbnail.heightAnchor.constraint(equalTo: thumbnail.widthAnchor)
+        ])
+    }
+}
+
+extension ProductGridCell {
+    func updateWithItem(_ newItem: Product) {
+        guard item != newItem else { return }
+        item = newItem
+        setNeedsUpdateConfiguration()
+    }
```

Comment:
이미 newItem이라는게 매개변수 레이블로 들어가있으니 아래와 같이 바꾸는게 나아보입니다.
```suggestion
    func update(_ newItem: Product) {
        guard item != newItem else { return }
        item = newItem
        setNeedsUpdateConfiguration()
    }
```

Reply:
조언해주신 것처럼 수정 진행했습니다!

[1e5067b](https://github.com/yagom-academy/ios-open-market/pull/152/commits/1e5067b1670d191e185f485b65285ddd4c6e072e)